### PR TITLE
[4.1] Update TinyMCE Skin Creator URL

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Update URL to go directly to the Skin Tool page.

### Testing Instructions
Go to System > Plugin
Edit `Editor - TinyMCE`
Click `Skin Creator` link.

Goes to the same page before/after PR.

### CODE REVIEW

File: administrator/language/en-GB/plg_editors_tinymce.ini [hunk 1]
```diff
@@ -49,7 +49,7 @@ PLG_TINY_FIELD_RESIZING_LABEL="Resizing"
 PLG_TINY_FIELD_SETACCESS_LABEL="Assign this Set to"
 PLG_TINY_FIELD_SKIN_ADMIN_LABEL="Administrator Skin"
 PLG_TINY_FIELD_SKIN_INFO_DESC="Copy your new skins to: /media/editors/tinymce/skins/ui."
-PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a href=\"http://skin.tiny.cloud\" target=\"_blank\">Skin Creator</a>"
+PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a href=\"https://skin.tiny.cloud/t5/\" target=\"_blank\">Skin Creator</a>"
```
Comment (PhilETaylor) [inline administrator/language/en-GB/plg_editors_tinymce.ini]: https://github.com/joomla/joomla-cms/pull/14881

```suggestion
PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a rel="noreferrer noopener" href=\"https://skin.tiny.cloud/t5/\" target=\"_blank\">Skin Creator</a>"
```

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on e5d2fef12af5994d483eefb45cea1ff1e2493b1a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36180">issues.joomla.org/tracker/joomla-cms/36180</a>.</sub>

File: administrator/language/en-GB/plg_editors_tinymce.ini [hunk 2]
```diff
@@ -49,7 +49,7 @@ PLG_TINY_FIELD_RESIZING_LABEL="Resizing"
 PLG_TINY_FIELD_SETACCESS_LABEL="Assign this Set to"
 PLG_TINY_FIELD_SKIN_ADMIN_LABEL="Administrator Skin"
 PLG_TINY_FIELD_SKIN_INFO_DESC="Copy your new skins to: /media/editors/tinymce/skins/ui."
-PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a href=\"http://skin.tiny.cloud\" target=\"_blank\">Skin Creator</a>"
+PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a rel="noreferrer noopener" href=\"https://skin.tiny.cloud/t5/\" target=\"_blank\">Skin Creator</a>"
```
Comment (Quy) [inline administrator/language/en-GB/plg_editors_tinymce.ini]:
```suggestion
PLG_TINY_FIELD_SKIN_INFO_LABEL="For customised skins go to: <a href=\"https://skin.tiny.cloud/t5/\" target=\"_blank\" rel=\"noopener noreferrer\">Skin Creator</a>"
```

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on 163531e80c37f3aaf8ad80d6b9ea7cf62d5c51a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36180">issues.joomla.org/tracker/joomla-cms/36180</a>.</sub>

**Comment by Quiviro** [comment]
I have tested this item :white_check_mark: successfully on 163531e80c37f3aaf8ad80d6b9ea7cf62d5c51a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36180">issues.joomla.org/tracker/joomla-cms/36180</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36180">issues.joomla.org/tracker/joomla-cms/36180</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**